src/components/new-request/search-components/business-lookup-fetch.vue

```diff
@@ -140,12 +141,14 @@ export default class BusinessLookupFetch extends Mixins(CommonMixin, SearchMixin
         const corpType = this.getSearchBusiness?.legalType as unknown as CorpTypeCd
         this.setEntityTypeCd(this.corpTypeToEntityType(corpType))
       } else {
-        this.setEntityTypeCd(this.getSearchBusiness.legalType)
+        this.setSearchCompanyType(CompanyTypes.NAMED_COMPANY)
+        this.setLocation(Location.BC)
+        this.setEntityTypeCd(this.getSearchBusiness?.legalType)
       }
 
       if (this.isChangeNameXpro) {
         this.setLocation(Location.CA)
-        this.setEntityTypeCd(this.getSearchBusiness.legalType)
+        this.setEntityTypeCd(this.getSearchBusiness?.legalType)
       }
```

Comment:
Was throwing error before. Fix for the bug. Same at line 146

Reply:
Please change both of these to `this.getSearchBusiness?.legalType || null`.

This is because conditional chaining can return "undefined", which is not reactive.